### ISSUE TITLE
[Feature Request] Go to post directly by pasting link

### ISSUE THREAD

**Issue by cantunborn** [Docile-Alligator/Infinity-For-Reddit#264]
This may be a fringe use case but there seems to be no way to go to a post or section (like multireddit or user profile) by pasting a link somewhere. Relevant results are not visible in search too.

**Comment by Docile-Alligator** [comment]
This is good idea!!!!

**Comment by yephny** [comment]
I agree, this is possible in Newpipe and it makes opening a Youtube link that you copied much easier.

**Comment by Docile-Alligator** [comment]
Yes.
